registration-processor/mosip-stage-executor/src/main/java/io/mosip/registration/processor/stages/executor/config/StagesConfig.java

```diff
@@ -104,7 +104,11 @@ private Map<String, Object> getConfigurationProperties() {
 			});
 			return (Map<String,Object>)configLoader.get().mapTo(Map.class);
 		}
-		// TODO change exception to catch and rethrow intruppted exception before base exception
+		catch(InterruptedException e) {
+			regProcLogger.error(ExceptionUtils.getStackTrace(e));
+			Thread.currentThread().interrupt();
+			return null;
```

Comment:
why returning null instead of throwing exception

Reply:
updated in latest commit